kratos/modeler/edge_swapping_2d_modeler.h

```diff
@@ -183,13 +183,13 @@ class EdgeSwapping2DModeler : public Modeler
             const int number_of_nodes = rThisModelPart.NumberOfNodes();
             const int number_of_elements = rThisModelPart.NumberOfElements();
 
-            #pragma omp parallel for
-            for(int i = 0 ; i < number_of_nodes ; i++)
-                nodes_array[i]->SetId(i+1);
+            IndexPartition<std::size_t>(number_of_nodes).for_each([&nodes_array](std::size_t Index){
```

Comment:
please add the include `parallel_utilties.h`!
this should be in every file that uses it!